ghost/admin/app/components/gh-member-avatar.hbs

```diff
@@ -1,7 +1,9 @@
 <figure class="gh-member-gravatar {{@containerClass}}">
-    <div class="gh-member-initials flex items-center justify-center br-100 {{@containerClass}}" style={{this.backgroundStyle}}>
-        <span class="gh-member-avatar-label {{or @sizeClass "gh-member-list-avatar"}}">{{this.initials}}</span>
-    </div>
+    {{#unless this.avatarImage}}
```

Comment:
I think this would be more clearly expressed an else after the `{{#if this.avatarImage}}` block

Reply:
Hi @ErisDS,
thank you for your feedback. I removed the `unless` and replaced that with an `else`.
Regards